lib/que/job.rb

```diff
@@ -94,13 +94,17 @@ def enqueue(*args)
           end
         end
 
+        tags = job_options[:tags] || []
+        data = { tags: tags }.merge(job_options[:data] || {})
```

Comment:
With this implementation you'll always get a tags key and empty array in the situation where there are no tags.

Perhaps something like this instead?
```ruby
tags = job_options[:tags] ? { tags: job_options[:tags] } : {}
data = tags.merge(job_options[:data] || {})
```